sklearn/tree/_export.py

```diff
@@ -745,6 +747,9 @@ def export_graphviz(decision_tree, out_file=None, *, max_depth=None,
         Number of digits of precision for floating point in the values of
         impurity, threshold and value attributes of each node.
 
+    fontname : str, default='helvetica'
+        Name of font used to render text.
```

Comment:
The code ignores `fontname` if `rounded` is false:

```suggestion
        Name of font used to render text. Only used when `rounded=True`
```

This is strange to me. I would consider removing the restriction, so that that fontname is set regardless of `rounded`.

Reply:
>This is strange to me. I would consider removing the restriction, so that that fontname is set regardless of rounded.

Later I would do some tests for removing this.

Reply:
Well, it still working after removing the restriction, just the shape of the nodes was changed.